docs/en_us/platform_api/source/profile_images/overview.rst

```diff
@@ -0,0 +1,24 @@
+################################################
+Profile Images Overview
+################################################
+
+Use the Profile Images API to upload or remove profile images.
+
+The requesting user can upload or remove his or her own profile image.
+
+User's with staff access can remove profile images from others' user accounts.
```

Comment:
I found it hard to parse this. Maybe just say "can remove profile images from any account".